### PR TITLE
Prepare release 13.3.0

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -25,6 +22,13 @@ Added
 
 Fixed
 -----
+- Fix manager shutdown in the test runner. If an unrecoverable exception
+  occurred while running a test, and never got caught (e.g. an unpicklable
+  exception in a parallel test worker), the listener would not get terminated
+  properly, leading to a hang.
+- Switch channels_redis dependency to upstream version
+- Data and collection name filters were fixed to work as expected (ngrams was
```
Comment (dblenkus) [inline docs/CHANGELOG.rst]: API filters

File: docs/CHANGELOG.rst [hunk 2]
```diff
@@ -25,6 +22,13 @@ Added
 
 Fixed
 -----
+- Fix manager shutdown in the test runner. If an unrecoverable exception
+  occurred while running a test, and never got caught (e.g. an unpicklable
+  exception in a parallel test worker), the listener would not get terminated
+  properly, leading to a hang.
+- Switch channels_redis dependency to upstream version
```
Comment (dblenkus) [inline docs/CHANGELOG.rst]: This should go under changes.

File: docs/CHANGELOG.rst (hunk 1 above)
Comment (mstajdohar) [reply docs/CHANGELOG.rst]: fixed

File: docs/CHANGELOG.rst (hunk 2 above)
Comment (mstajdohar) [reply docs/CHANGELOG.rst]: fixed

**Comment by mstajdohar** [comment]
Thanks @dblenkus

**Pull request merged**